torch/_inductor/codegen/triton.py

```diff
@@ -822,10 +851,10 @@ def codegen_header(self, code, no_x_dim=False):
         if self.is_loop():
             code.writeline(f"{self.name} = {x}offset + {x}base")
         elif x == "r" and self.kernel.persistent_reduction:
-            # no need to "roffset = "
             code.writeline(
                 f"{self.name} = {self.ranges_code()}",
             )
+            code.writeline("roffset = 0")
```

Comment:
Curious why we add this even though roffset will be setup properly in the reduction loop

Reply:
For persistent reductions